POLICY.md

```diff
@@ -6,10 +6,8 @@ Entries are accepted if they meet the following criteria:
 
 * Entries must respect the [IPFS Community Code of Conduct](https://github.com/ipfs/community/blob/master/code-of-conduct.md)
 * Entries must use *IPFS* (libp2p, ipld, multiformats is not enough)
+* Entries in the **apps** & **tools** categories must be **open-source** and contain a link to their source code.
 * Entries must be immediately usable (eg. nothing that is paywall-only, even a free trial would be ok)
 * Entries must have a clear license and/or privacy policy where applicable
 
-This policy is evaluated by the site admins upon submission, per the process defined in the [contribution guide](CONTRIBUTING.md).
-
-This policy was developed [here](https://github.com/ipfs/awesome-ipfs/issues/150) - please add questions, comments and suggestions there.
```

Comment:
Should we link to this PR in that issue and mention that there was an update to policy?

Reply:
Sure! Just did it: https://github.com/ipfs/awesome-ipfs/issues/150#issuecomment-1729602722